### PR TITLE
added get_latest_tag in install.sh

### PR DESCRIPTION
**Pull request by maheshmeleti**
## 📚 Description

Remove downtime on install latest #225

## 🔖 Changes

1. Updated install.sh to fetch the latest version from the git hub instead of hard coding.

2. Updated RELEASE.md file.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -2,13 +2,23 @@
 
 # shellcheck disable=SC2164
 # shellcheck disable=SC2103
+declare -r BASHUNIT_GIT_REPO="https://github.com/TypedDevs/bashunit"
+
+function get_latest_tag() {
```
Comment (Chemaclass) [inline install.sh:7]: Can you write a test to ensure the behaviour of this new function?

**Review by Chemaclass — COMMENTED** [review]
Thanks for your PR! Although, I am missing some tests for this new logic. Some ideas: an acceptance test to make sure the whole thing works as expected, and/or a unit test for the new little function behaviour.

File: install.sh (hunk 1 above)
Comment (maheshmeleti) [reply install.sh:7]: I am using the existing function for the latest update; 
A test case already exists to test the function test_get_latest_tag() in \bashunit\tests\unit\helpers_test.sh.

**Review by khru — APPROVED** [review]
Looks good

**Review by Chemaclass — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (Chemaclass) [reply install.sh:7]: You are right, that test is covering that area as well, thanks!

**Review by antonio-gg-dev — APPROVED** [review]

**Comment by antonio-gg-dev** [comment]
Closes #225

**Pull request merged**